policy/modules/services/rpc.te

```diff
@@ -325,12 +326,14 @@ dev_rw_lvm_control(nfsd_t)
 
 files_getattr_tmp_dirs(nfsd_t)
 files_manage_mounttab(nfsd_t)
+files_search_mnt(nfsd_t)
```

Comment:
I added the access for `mnt_t` since it was specific to the AVC I ran into on the system I developed this on, but I agree all mountpoints is probably better.